WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -477,6 +481,7 @@ public boolean onPrepareOptionsMenu(Menu menu) {
                         } else {
                             saveMenuItem.setTitle(R.string.update_verb);
                         }
+                        mSaveWithoutLeaving = true;
```

Comment:
Hard to weigh in here since I'm not 100% filled in on the thinking behind it, but generally, if you're editing a post and the save action does not publish the post, I agree that you want to remain in the editor.